Dockerfile

```diff
@@ -0,0 +1,88 @@
+
+# This is a Dockerfile that extends an installation of itk and modules
+#in the dsarchive/histomicstk:v0.1.3 
+
+
+FROM dsarchive/histomicstk:v0.1.3
+
+MAINTAINER Bilal Salam <bilal.salam@kitware.com>
+
+
+RUN apt-get update && \
+    apt-get install -y git wget python-qt4\
+    openslide-tools python-openslide \
+    build-essential \
+    swig \
+    make \
+    zlib1g-dev \
+    curl \
+    libcurl4-openssl-dev \
+     libexpat1-dev \
+    unzip && \
+    apt-get autoremove && \
+    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+#install requirements for itk
+
+WORKDIR /
+
+#need cmake
+RUN git clone https://github.com/Kitware/CMake.git
+WORKDIR CMake 
+#cmake release v3.6.0
+RUN git checkout e31084e65745f9dd422c6aff0a2ed4ada6918805 && \
+./bootstrap --system-curl && make && make install
+#by default cmake does not use system ssl
```

Comment:
You can combine all commands to install cmake as follows
```Dockerfile
RUN cd / && git clone https://github.com/Kitware/CMake.git && cd CMake && \
    git checkout e31084e65745f9dd422c6aff0a2ed4ada6918805 && \
    ./bootstrap --system-curl && make && make install
```
That way all commands to install cmake will be in one cached docker layer.

Also test, if the [apt-get install](https://github.com/InsightSoftwareConsortium/ITKIOOpenSlide/blob/master/test/Docker/Dockerfile#L7) works for cmake. Alternatively, i think you can download the latest pre-complied version for linux using wget from [here](https://cmake.org/download/) and install it.